executor/show_test.go

```diff
@@ -609,6 +609,19 @@ func (s *testSuite5) TestShowCreateTable(c *C) {
 			"  `a` varchar(123) DEFAULT NULL\n"+
 			") ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin",
 	))
+
```

Comment:
Do we need to handle the SQL of `show keys from t;`? It seems that we need to add a field of `Expression`.

Reply:
Yeah, I will support this in following PR.